tests/foreman/performance/common/perf_candlepin.py

```diff
@@ -0,0 +1,116 @@
+"""Test utilities for writing Candlepin tests
+
+Part of functionalities of Candlepin are defined in this module
+and have utilities of single register by activation-key, single
+register and attach, single subscription deletion.
+
+"""
+
+
+from robottelo.common import conf, ssh
+
+import logging
+import time
+import requests
+
+
+class CandlepinPerfTest:
+    """Measures performance of RH Satellite 6
+
+    Candlepin Subscription functionality
+    """
+    username = conf.properties['foreman.admin.username']
+    password = conf.properties['foreman.admin.password']
+    serverhost = conf.properties['main.server.hostname']
+
+    # create logger
+    logger = logging.getLogger('robottelo')
+    logger.setLevel(logging.DEBUG)
+    requests.packages.urllib3.disable_warnings()
+
+    @classmethod
+    def single_register_activation_key(cls, ak_name, default_org, vm):
+        """Subscribe VM to Satellite by Register + ActivationKey"""
+
+        # note: must create ssh keys for vm if running on local
+        result = ssh.command('subscription-manager clean', hostname=vm)
+        result = ssh.command('time -p subscription-manager register \
+                             --activationkey={0} --org={1}'
+                             .format(ak_name, default_org), hostname=vm)
+        if result.return_code != 0:
+            cls.logger.error(result.stderr)
+        else:
+            real_time = [real for real in result.stderr.split('\n')
+                         if real.startswith('real')]
+            real_time = real_time[0].split(' ')[1]
+            return float(real_time)
+
+    @classmethod
+    def single_register_attach(cls, sub_id, default_org, environment, vm):
+        """Subscribe VM to Satellite by Register + Attach"""
+
+        ssh.command(
```

Comment:
This does not fit one line `ssh.command('subscription-manager clean', hostname=vm)? If not is better to write:

```python
ssh.command(
    'subscription-manager clean', hostname=vm
)